Data/FileEmbed.hs

```diff
@@ -87,6 +89,26 @@ embedFile fp =
 #endif
   (runIO $ B.readFile fp) >>= bsToExp
 
+-- | Maybe embed a single file in your source code depending on whether or not file exists.
+--
+-- Warning: Does not use `qAddDependentFile` to force a recompile depending on changes to or appearance of the file.
```

Comment:
Sorry, I think I was unclear in my previous comment. If the file exists, it _should_ use `addDependentFile`. Only in the case of the file being absent should it skip that.

Reply:
> Also added a couple tests, hope that's ok.

Far more than OK, thank you!